package.json

```diff
@@ -756,6 +758,7 @@
   },
   "peerDependencies": {
     "@folio/stripes": "^6.0.0",
+    "ky": "^0.23.0",
```

Comment:
It is provided by stripes-core: https://github.com/folio-org/stripes-core/blob/a66794f3a182c01b5a1592027493f91d4f3d7509/package.json#L113

Reply:
So, I've previously relied on that functionality, but I'm not sure if I'm supposed to. @skomorokh told me recently that peerDeps being satisfied by something other than their parent packages is not really part of the API and it's flaky. I haven't had time to look into it though...

Reply:
Ah! By "that functionality", you don't mean anything related to ky specifically, but expressing a peer dependency on a regular dependency _of_ a peer dependency — correct?

If we're not supposed to do that — this is another of the areas where I have no idea what the rules are — I can certainly promote it to a regular dependency.

I know this is a hopelessly optimistic question to ask, but: are "the rules" documented anywhere?

Reply:
Use peer if it's something where everything needs to be on the same version. Peers can only be satisfied by ancestors so it'll be up to the platform to provide it. So, eg. if you're using ky to hit some other webservice and using okapiKy but nothing you're doing requires that both have exactly the same version, you should prefer to use a dependency so no one needs to think about keeping ky in sync across repos.

Reply:
> Ah! By "that functionality", you don't mean anything related to ky specifically, but expressing a peer dependency on a regular dependency of a peer dependency — correct?

Right.

I couldn't find something published by NPM officially quickly, but this bit talks about how it needs to be [satisfied up the dependency chain](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#peerdependencies). 

Additionally, the naming of it seems like a hint to me as well. A peer is a package at the same level. So `react` and `ui-users` are peers because both are dependencies of the platform. But `ky` isn't a peer because it's a level lower. It's not a sibling of `react` or `ui-users`, but rather....a niece?

Reply:
Thanks to you both. I have fixed this in https://github.com/folio-org/ui-inventory/pull/1294/commits/4c49300949e60cd45fa03ff20b849782b6c028bd